### PR TITLE
List Ubuntu 18.04 as being supported

### PR DESCRIPTION
**Pull request by morphis**

### CODE REVIEW

**Comment by adglkh** [comment]
LGTM

File: requirements.md [hunk 1]
```diff
@@ -23,8 +23,11 @@ You must run either the [server](https://ubuntu.com/download/server) or the [clo
 
 Supported Ubuntu versions:
 
+* 18.04 (bionic)
 * 20.04 (focal)
 
+For new deployments Ubuntu 20.04 (focal) should be preferred.
```
Comment (ru-fu) [inline requirements.md:29]:
```suggestion
For new deployments, Ubuntu 20.04 (focal) is preferred.
```

**Review by ru-fu — COMMENTED** [review]
Should we clarify that you need to update the kernel for 18.04 and point to the next section?

**Review by aguadoenzo — APPROVED** [review]

**Comment by ru-fu** [comment]
@morphis I added a commit to point out the kernel limitation. OK for you?

**Comment by morphis** [comment]
@ru-fu Sorry, I completely forgot about that part. Yes, looks good to me.

**Pull request merged**